source/test/TestDirector.cc

```diff
@@ -147,6 +154,11 @@ void TestDirector::test(const TestData &data) {
     case Changelog::AddedExtVel:
         checkDesync(data.extVel, extVel, "extVel");
         [[fallthrough]];
+    case Changelog::AddedCheckpoints:
+        checkDesync(data.raceCompletion, raceCompletion, "raceCompletion");
+        checkDesync(data.checkpointId, checkpointId, "checkpointId");
+        checkDesync(data.jugemId, jugemId, "jugemId");
+        [[fallthrough]];
```

Comment:
The STATUS.md bug is caused by incorrect positioning of this switch case. The extra frame is caused by desyncs happening *above* this case, and the resulting desync a frame later is caused by the resultant desync in the position and full rotation.

This needs to be moved above `Changelog::AddedRotation` to work correctly, and the STATUS.md change can be removed.